### PR TITLE
Add HA cluster checkbox button on sizing page

### PR DESCRIPTION
**Pull request by arbulu89**
This PR implements the HA enabled checkbox in the cluster sizing page.
Check the README file to understand how to use it.
ha_enabled

### CODE REVIEW

File: config/locales/en.yml [hunk 1]
```diff
@@ -84,6 +84,8 @@ en:
     wrapup: "Next steps after deployment"
 
   page_subtitle:
+    hana_ha_enabled_info: |
+      Enable HA clustering and System Replication
```
Comment (MalloZup) [inline config/locales/en.yml:88]: I was thinking that `clustering` word might redundant here.

` Enable HA and System Replication`

File: README.md [hunk 2]
```diff
@@ -94,9 +94,10 @@ Variables **must** be defined in terraform JSON format, and named `variable*.tf.
 
 The following variables will not be displayed on the variable entry form, but will be populated via other application interfaces:
 - `instance_type`: the virtual machine type to be used when starting cloud instances; this will be populated from the _Size Cluster_ page.
-- `instance_count`: the number of virtual machines to be deployed; this will be populate from the _Size Cluster_ page.
+- `instance_count`: the number of virtual machines to be deployed; this will be populated from the _Size Cluster_ page.
 - `region`: the cloud provider's region where services will be established. If _blue-horizon_ is run in a cloud environment; the location will be autodetected via Instance Meta Data Services (IMDS).  
   ⚠ _End users should be notified that the application needs to run in the same region where it will be deployed._
+- `hana_ha_enabled`: checkbox to enable/disable HA capabilities in the HANA cluster; this will be populated from the _Size Cluster_ page.
```
Comment (MalloZup) [inline README.md]: As habit  I think is always good to specify the `default` behaviour when documenting a variable.

In our case here is `false`.

**Review by MalloZup — APPROVED** [review]

File: app/models/cluster.rb [hunk 3]
```diff
@@ -81,5 +87,8 @@ def save!
     prefixed_set(:instance_type, @instance_type)
     prefixed_set(:instance_type_custom, @instance_type)
     prefixed_set(:instance_count, @instance_count)
+    prefixed_set(
```
Comment (MalloZup) [inline app/models/cluster.rb:90]: compared to other code, this reads ugly. 
I searched a bit it seems we need to convert ths to a true/false variable.

File: config/locales/en.yml (hunk 1 above)
Comment (arbulu89) [reply config/locales/en.yml:88]: It is not redundant. `HA clustering` stand for `High availability clustering`. `HA` by itself might not mean anything about clustering.

File: app/models/cluster.rb (hunk 3 above)
Comment (arbulu89) [reply app/models/cluster.rb:90]: This is how it is used in other parts of the code

File: README.md (hunk 2 above)
Comment (arbulu89) [reply README.md]: It is not `false`. This default value is the value we give to `hana_ha_enabled` variable in the `variables.tf` file. Anyway, I will add more information to clarify this

**Pull request merged**